sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -78,6 +72,12 @@
 or a table. Pipeline construction will fail with a validation error if neither
 or both are specified.
 
+When reading from BigQuery using `BigQuerySource`, bytes are returned as
```

Comment:
Can we add some guidance when to use which?

Reply:
Ping on this - do we need to guide users in any way which transform they should use? Is some of them experimental or to be deprecated?